coriolis/tasks/osmorphing_tasks.py

```diff
@@ -72,7 +124,14 @@ def _run(self, ctxt, instance, origin, destination, task_info,
             instance_script,
             event_handler)
 
-        return {}
+        volumes_info = instance_deployment_info.get('volumes_info', [])
+        LOG.debug('Volumes info before root disk reordering: %s', volumes_info)
+        _reorder_root_disk(volumes_info, osmorphing_info.get('os_root_dev'),
+                           os_type)
+        LOG.debug('Volumes info after root disk reordering: %s', volumes_info)
+
+        return {
+            'instance_deployment_info': instance_deployment_info}
```

Comment:
Simply putting the `volumes_info` back into the `instance_deployment_info` and hoping that the plugins will handle it isn't a safe way to go about things...

Please make the `OSMorphingTask` both require and return the `volumes_info` and manipulate it directly instead.
Out goal should be to not have to care/check on any of the plugins following this patch.